### ISSUE TITLE
Accessibility Issue in High Contrast Mode - The keyboard focus is not visible for the Add Question button

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#3648]
Original issue: [T11196 - Accessibility issue: The keyboard focus is not visible on "Add Question button" after applying high contrast aquatic/desert mode](https://surveyjs.answerdesk.io/internal/ticket/details/T11196).
![HighContrastModeAddButton](https://user-images.githubusercontent.com/22372972/200767236-7ae48f83-bd4c-400e-bc29-83a3b0f1da2f.gif)
